src/test/java/programminglife/gui/GuiControllerTest.java

```diff
@@ -198,7 +198,10 @@ public void clickOnTest() {
         }
 
         sleep(5, TimeUnit.SECONDS);
+        //backspace to remove the initial values.
+        clickOn("#txtMaxDrawDepth").type(KeyCode.BACK_SPACE);
```

Comment:
actually it did delete everything instead of only 1 number, but I am not sure how it worked.